packages/css/src/shorthand-parser/index.ts

```diff
@@ -4,6 +4,10 @@ import { tokenizeValue } from "./value-tokenizer";
 const unitMatch = /^[0-9.]+[a-z|%]/;
 const easingMatch = /\(.*\)|ease|ease-in|ease-out|ease-in-out|linear|step-start|step-end/;
 
+const fontSizeMatch = /^([+-]?[0-9.]+([a-z]+|%)?|large(r)?|medium|small(er)?|x{1,3}-large|x{1,2}-small)(\/[+-]?[0-9.]+([a-z]+|%)?)?$/;
+const fontStyleMatch = /^[+-]?[0-9.]+deg$/;
+const fontWeightMatch = /^(0*[1-9][0-9]{0,2}|1000|bold(er)?|lighter)$/;
```

Comment:
These regular expressions are intended to follow the patterns seen in `unitMatch` and `easingMatch`.

A more accurate regular expression for CSS numbers may be `/([+-]?(?:\d+(?:\.\d*)?|\.\d+)(?:[Ee][+-]?\d+)?)/`. However, if parsing performance is of importance, it may be _even better_ to identify numbers at the tokenization stage and leave out these regular expressions altogether.

Reply:
Yeah I totally agree with that 👍 
the tokenizer used to support this functionality, it would return an array of tokens with types belonging to 
```string | number | number with unit | CSS function```
but we weren't sure at the time how we were going to use it so we kept it as simple as possible but now i'm thinking we should bring this functionality back.

we can change this when we update the tokenizer but for now, this is good